BUILD.rst

```diff
@@ -26,44 +26,10 @@ test/
 
 Unit and Acceptance Tests
 -------------------------
+To run tests give command::
+	python test/run_tests.py chomre
```

Comment:
- chomre -> chrome
- If `run_tests.py` is executable, could just use `test/run_tests.py chrome`
- Should it rather be something generic like `<browser>` instead of `chrome`?